assets/queries/dockerCompose/network_mode_host_exists/metadata.json

```
@@ -0,0 +1,10 @@
+{
+    "id": "071a71ff-f868-47a4-ac0b-3c59e4ab5443",
+    "queryName": "Network Mode Host Exists",
```

```suggestion
    "queryName": "Shared Host Network Namespace",
```

Please, also update the query folder name